test/e2e/run_test.go

```diff
@@ -1376,7 +1376,26 @@ USER mail`
 	})
 
 	It("podman run --tz", func() {
-		session := podmanTest.Podman([]string{"run", "--tz", "foo", "--rm", ALPINE, "date"})
+		testDir := filepath.Join(podmanTest.RunRoot, "tz-test")
+		err := os.MkdirAll(testDir, 0755)
+		Expect(err).To(BeNil())
+
+		tzFile := filepath.Join(testdir, "tzfile.txt")
+		file, err := os.Create(tzFile)
+		Expect(err).To(BeNil())
+		defer file.Close()
+
+		_, err = file.WriteString("Hello")
+		Expect(err).To(BeNil())
+
+		session := podmanTest.Podman([]string{"run", "--tz", ("../../../" + tzFile), "--rm", ALPINE, "date"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Not(Equal(0)))
```

Comment:
Can you check for the correct error message here with `Expect(session.ErrorToString()).To(ContainSubstring("error finding timezone for container"))`. Otherwise the non zero exit code could come from the date command.